egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -28,6 +28,8 @@ def ParseSpliceString(splice_indexes, label_delay=None):
     if label_delay is not None:
         left_context = -label_delay
         right_context = label_delay
+    if preserve_state == "true":
```

Comment:
Here left_context is used to decide the left boundary of a chunk. If there is only one "output", the left boundary will be max(0, 2 - label_delay) = max(0, 2 - 5)=0. Now we have state outputs that are not label-delayed, so the left boundary of a chunk will be max(0, 2)=2.